### PR TITLE
[JENKINS-62489] Support m6g ec2 instances

### PR DESCRIPTION
**Pull request by schottsfired**
Picks up https://github.com/jenkinsci/aws-java-sdk-plugin/releases/tag/aws-java-sdk-1.11.821

### CODE REVIEW

**Comment by schottsfired** [comment]
My keyboard went bananas and hit enter before I finished the title & description 🤦

**Comment by xsacha** [comment]
Does this also add support for c5a instance?

**Comment by schottsfired** [comment]
Hi @xsacha, it should, since c5a instances are in `1.11.796`: https://github.com/aws/aws-sdk-java/blob/master/CHANGELOG.md#111796-2020-06-04.

File: pom.xml [hunk 1]
```diff
@@ -83,10 +83,10 @@ THE SOFTWARE.
     <properties>
         <revision>1.51</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.204.6</jenkins.version>
         <java.level>8</java.level>
         <jcasc.version>1.36</jcasc.version>
-        <aws-java-sdk.version>1.11.723</aws-java-sdk.version>
+        <aws-java-sdk.version>1.11.821</aws-java-sdk.version>
```
Comment (batmat) [inline pom.xml:89]: After a second thought, I realize: this is going to force this plugin to require this release with new additions, which is good. 

But this was not strictly necessary for a Jenkins instance to be able to see the new types: one would just have to make sure to update to the latest `aws-java-sdk` Jenkins plugin (i.e. 1.11.821+) right?

**Pull request merged**